crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -732,6 +737,17 @@ where
                             on_plotted_sector_callback(plotted_sector, old_plotted_sector);
                             farmer_metrics.observe_sector_plotting_time(&single_disk_farm_id, time);
                             farmer_metrics.sector_plotted.inc();
+                            if let Some(_) = &old_plotted_sector {
+                                farmer_metrics.update_farm_replotted(&single_disk_farm_id, 1);
+                            } else {
+                                farmer_metrics.update_farm_plotted(&single_disk_farm_id, 1);
+                            }
+                        }
+                        SectorUpdate::Expiration(SectorExpirationDetails::AboutToExpire) => {
+                            farmer_metrics.update_farm_expired(&single_disk_farm_id, 1);
```

Comment:
I think we cannot count the Expired and AboutToExpire sectors separately using the incremental counters only, because it isn't possible to determine which sector is replotted within the sector update callback fn. So, I decided to count them together.

Reply:
We don't need to know "which" sector, we only need to know "how many" and since fully expired sectors are being replotted before those that are about to expire, we already know what to decrement.

Reply:
I initially wanted to count expirations and replots cumulatively in the corresponding counter metrics and calculate the difference (actual expired sectors) outside of the farmer, this approach provides a bit more stored data. If you prefer storing only the differences, I can of course change the scheme.

Reply:
I see. I guess that would work as well, but I'm not sure what would be more flexible. We already have number of sectors plotted in total with metrics before this PR. I'm open for options either way if there is some additional information to be gained from it.